lib/lotus/helpers/numbers.rb

```diff
@@ -0,0 +1,17 @@
+require 'lotus/helpers/numbers/number_parser'
+
+module Lotus
+  module Helpers
+    # TODO
+    #
+    module Numbers
+      private
+      # TODO
+      #
+      def format_number(number, options = {})
+        number_parser = NumberParser.new(number, options)
```

Comment:
Why the implementation of `format_number` is delegated to `NumberParser`. Isn't better to call it `NumberFormatter`?